File: EntityFX.NetBenchamarks.Core/Generic/StringManipulation.cs
using System.Diagnostics;

namespace EntityFX.NetBenchmark.Core.Generic
{
    public class StringManipulation : StringManipulationBase, IBenchamrk
    {
        public override BenchResult Bench()
        {
            var sw = new Stopwatch();
            sw.Start();
            var str = "the quick brown fox jumps over the lazy dog";
            string str1;
            for (int i = 0; i < Iterrations; i++)
            {
                str1 = DoStringManipilation(str);
            }
            return BuildResult(sw);
        }
    }
}

File: src/python/entityfx/string_manipulation.py
from entityfx.string_manipulation_base import StringManipulationBase

class StringManipulation(StringManipulationBase):
    
    def benchImplementation(self) -> str:
        str0_ = "the quick brown fox jumps over the lazy dog"
        str1 = ""
        i = 0
        while i < self._iterrations: 
            str1 = StringManipulationBase._doStringManipilation(str0_)
            i += 1
        return str1
